benchmark/scenarios/no_load.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
+"""
+No load scenario for the control service benchmarks.
+"""
+
+from zope.interface import implementer
+
+from twisted.internet.defer import Deferred, succeed
+
+from .._interfaces import IScenario
+
+
+@implementer(IScenario)
+class NoLoadScenario(object):
```

Comment:
This appears to be untested. I realize it's very simple, but because our static checks are so limited, it really is worth having direct coverage so we can be sure we're avoiding common mistakes. Please write some tests.